price_feed_poller.py

```diff
@@ -195,10 +199,12 @@ def log_loop(
             # Result not ready. Wait for following group
             print("Waiting in contract %s for DR Result #%d" % (element["feed"].address, element["currentId"]))
         else:
-          # Check ellapsed time since last request to this feed contract:
+          if timestamps[index] == 0:
+            timestamps[index] = element["lastTimestamp"]
```

Comment:
It would be nice to log `lastTimestamp` (previously converted to readable UTC datetime), if `timestamps[index]` gets initialized to a value different than zero.

Reply:
Something like "Price feed last updated at timestamp x" would be nice